xclim/__init__.py

```diff
@@ -89,7 +91,9 @@ def __build_icclim(mode='warn'):
                'DTR': indices.daily_temperature_range,
                }
 
-    return build_module('icclim', mapping, doc="""ICCLIM indices""", mode=mode)
+    mod = build_module('icclim', mapping, doc="""ICCLIM indices""", mode=mode)
+    sys.modules['xclim.icclim'] = mod
+    return mod
```

Comment:
Why is this building a module? I've never seen this before and am curious. What's gained by this behaviour?

Reply:
Woops, TEDLT. Approved.